src/LiveComponent/src/Test/TestLiveComponent.php

```diff
@@ -51,13 +51,25 @@ public function __construct(
             $this->metadataFactory->getMetadata($mounted->getName())
         );
 
-        $this->client->request('GET', $this->router->generate(
-            $this->metadata->get('route'),
-            [
-                '_live_component' => $this->metadata->getName(),
-                'props' => json_encode($props->getProps(), flags: \JSON_THROW_ON_ERROR),
-            ]
-        ));
+        if ('POST' === $method = strtoupper($this->metadata->get('method'))) {
+            $this->client->request(
+                $method,
```

Comment:
We can avoid the `$method` entirely and put `POST` here and `GET` below. Those are the only 2 valid methods.

Reply:
Done.